connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSourceTask.java

```diff
@@ -409,6 +410,9 @@ private boolean sendRecords() {
     // RegexRouter) topic creation can not be batched for multiple topics
     private void maybeCreateTopic(String topic) {
         if (!topicCreation.isTopicCreationRequired(topic)) {
+            log.trace("The topic creation setting is disabled or the topic name {} is already in the topic cache." +
```

Comment:
This is going to be very repetitive after the first time during which the topic might be created. 

What's the hint we are trying to extract here?

Reply:
@kkonstantine , thanks for the comment, what I'm trying to do here is to let the user know if the creating topic pending for a long time without response (default timeout is 30 or 60 secs), we let user know earlier that this situation might happen, and it could because the `auto.create.topics.enable` is disabled. Do you think it's necessary?